config/locales/application_form.yml

```diff
@@ -317,6 +317,10 @@ en:
         - Fourth referee
         - Fifth referee
         - Sixth referee
+        - Seventh referee
+        - Eighth referee
+        - Ninth referee
+        - Tenth referee
```

Comment:
Can we replace this by the TextOrdinalizer?

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/services/text_ordinalizer.rb#L1